src/store/wallet.js

```diff
@@ -483,14 +482,20 @@ export default {
       }
     },
     subscribeToWalletNotifications: async function(context) {
-      context.state.api.subscribeToNotifications(
+      await context.state.api.subscribeToNotifications(
         { session_id: this.state.wallet.sessionId },
-        notification =>
-          context.commit('setBalance', {
-            balance: { total: notification[0].accountBalance.amount },
-          })
+        ([notification]) => {
```

Comment:
What do you think about  add a callback to flatten [notification] i in `subscribeToNotifications` to simplify the api? I add a suggestion for that in `subscribeToWalletNotifications`.